src/app/layouts/auth-layout/auth-layout.module.ts

```diff
@@ -1,22 +1,20 @@
-import {NgModule} from '@angular/core';
-import {CommonModule} from '@angular/common';
-import {HttpClientModule } from '@angular/common/http';
-import {RouterModule} from '@angular/router';
-import {FormsModule, ReactiveFormsModule} from '@angular/forms';
-import {NgxSpinnerModule} from 'ngx-spinner';
-
-import {AuthLayoutRoutes} from './auth-layout.routing';
-import {LoginComponent} from '../../pages/login/login.component';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { RouterModule } from '@angular/router';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NgxSpinnerModule } from 'ngx-spinner';
+import { AuthLayoutRoutes } from './auth-layout.routing';
+import { LoginComponent } from '../../pages/login/login.component';
 import { LoginFormComponent } from '../../pages/login/login-form/login-form.component';
```

Comment:
Please use absolute imports ... Relative imports are really hard to read and hard to edit should there be changes required later on